packages/jsii-pacmak/lib/targets/go/runtime/method-call.ts

```diff
@@ -62,6 +70,13 @@ export class MethodCall {
     );
   }
 
+  private get implMapVar(): string {
+    return slugify(
+      'implMap',
+      this.parent.parameters.map((p) => p.name),
+    );
+  }
```

Comment:
Looks like this would be invoked a bunch of times and could be kinda slow & thrash's... Maybe store the result in a private and re-use?